code/modules/mob/dead/observer/observer.dm

```diff
@@ -323,10 +323,23 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 		return
 	GLOB.ghost_hud_panel.ui_interact(src)
 
-/mob/dead/observer/proc/toggle_all_huds_on()
+/**
+ * Toggles on all HUDs for the ghost player.
+ *
+ * Enables antag HUD only if the ghost belongs to an admin.
+ *
+ * Arguments:
+ * * user - a reference to the ghost's old mob. This argument is required since `src` does not have a `client` at this point.
```

Comment:
Should this be changed to "A" rather than "a"? Genuine question since I can't remember what the standard is.

Reply:
I'm not sure, but I always just keep it lowercase.

Reply:
> 
> 
> Should this be changed to "A" rather than "a"? Genuine question since I can't remember what the standard is.

In terms of grammar, the `-` There indicates the start of a new sentence. So yes, it should technically be capitalised.